operator/gitops/argocd/pipeline-service/openshift-pipelines/chains-observability-service.yaml

```diff
@@ -5,13 +5,16 @@ metadata:
   name: tekton-chains
   namespace: openshift-pipelines
```

Comment:
Could you also add argo sync wave for consistency? Although argocd will add a dfault annotation with sync-wave 0, this is something we agreed upon for having explicit annotation. Specifically, 
```
  annotations:
    argocd.argoproj.io/sync-wave: "1"
```

Reply:
https://github.com/openshift-pipelines/pipeline-service/pull/694